Misc/NEWS.d/next/Library/2025-11-22-16-33-48.gh-issue-141863.4PLhnv.rst

```diff
@@ -0,0 +1,2 @@
+Update :ref:`asyncio-streams` to use :gh:`139871` :func:`bytearray.take_bytes`
+for a over 10% performance improvement on pyperformance asyncio_tcp benchmark.
```

Comment:
nit, the gh reference is unnecessary, just the link to take_bytes seems sufficient to me